docs/00-initial-setup/README.md

```diff
@@ -454,11 +454,14 @@ In addition to the lambda code, the configurations for Lambda function and the R
 	To do it from commandline:
 
 	```
-	aws cloudformation describe-stacks --region $REGION --stack-name CustomizeUnicorns --query "Stacks[0].Outputs[0].OutputValue"
+	aws cloudformation describe-stacks --region $REGION --stack-name CustomizeUnicorns --query "Stacks[0].Outputs[0].OutputValue --output text"
```

Comment:
The closing double-quote for the --query is in the wrong location on this line, causing an error in the workshop. Should be aws cloudformation describe-stacks --region $REGION --stack-name CustomizeUnicorns --query "Stacks[0].Outputs[0].OutputValue" --output text